Graph type update for Cypher 5 Constraints (#1395)
neo4j/docs-cypher: modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

* link:https://neo4j.com/docs/cypher-manual/25/introduction/[Cypher 25 Manual]
====

[[cypher-deprecations-additions-removals-2025.xx]]

> == Neo4j 2025.xx

Reviewer: ```suggestion
[[cypher-deprecations-additions-removals-2026.02]]
== Neo4j 2026.02
```

* link:https://neo4j.com/docs/cypher-manual/25/introduction/[Cypher 25 Manual]
====

[[cypher-deprecations-additions-removals-2025.xx]]
== Neo4j 2025.xx

=== Updated in Cypher 5

[cols="2", options="header"]
|===
| Feature
| Details

a|
label:functionality[]
label:updated[]
[source, syntax, role="noheader"]
----
SHOW [NODE] EXIST[ENCE] CONSTRAINTS
----

> | The `NODE EXIST[ENCE]` and `EXIST[ENCE]` type filtering options allowed by the `SHOW CONSTRAINTS` command now also return node label existence constraints (crated as part of node element types in a link:{neo4j-docs-base-uri}/operations-manual/25/schema/graph-types/[graph type] (only available in Cypher 25)).

Reviewer: ```suggestion
| The `NODE EXIST[ENCE]` and `EXIST[ENCE]` type filtering options allowed by the `SHOW CONSTRAINTS` command now also return node label existence constraints (created as part of node element types in a link:{neo4j-docs-base-uri}/operations-manual/25/schema/graph-types/[graph type] (only available in Cypher 25)).
```

Reviewer 2: integrated into the suggestion below

> | The `NODE EXIST[ENCE]` and `EXIST[ENCE]` type filtering options allowed by the `SHOW CONSTRAINTS` command now also return node label existence constraints (crated as part of node element types in a link:{neo4j-docs-base-uri}/operations-manual/25/schema/graph-types/[graph type] (only available in Cypher 25)).

Reviewer: why does the link have `operations-manual` in it? 🤔 shouldn't it be cypher-manual?
```suggestion
| The `NODE EXIST[ENCE]` and `EXIST[ENCE]` type filtering options allowed by the `SHOW CONSTRAINTS` command now also return node label existence constraints (created as part of node element types in a link:{neo4j-docs-base-uri}/cypher-manual/25/schema/graph-types/[graph type] (only available in Cypher 25)).
```

For more information, see xref:constraints/syntax.adoc#constraints-syntax-list-type-filter[Syntax -> Type filters].

a|
label:functionality[]
label:updated[]
[source, syntax, role="noheader"]
----
SHOW CONSTRAINTS 
----

a| The `type` column returned by xref:constraints/list-constraints.adoc[`SHOW CONSTRAINTS`] can now return three additional constraint types: `NODE_LABEL_EXISTENCE`, `RELATIONSHIP_SOURCE_LABEL` and `RELATIONSHIP_TARGET_LABEL`.
These new constraint types are created as part of node and relationship element types when setting a graph type (only available in Cypher 25).

> For more information, see the link:{neo4j-docs-base-uri}/operations-manual/25/schema/graph-types/[Cypher 25 Manual -> Graph types].

Reviewer: ```suggestion
For more information, see the link:{neo4j-docs-base-uri}/cypher-manual/25/schema/graph-types/[Cypher 25 Manual -> Graph types].
```
